### PR TITLE
Html font size

### PR DESCRIPTION
**Pull request by vomikan**

### CODE REVIEW

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: 33aafe54d018a00a536c6385e44422e814f21ba6
  * Invalid type. Valid types are **feat, fix, docs, style, refactor, perf, test, chore, revert, cleanup, format**

Guidelines are available at https://github.com/moneymanagerex/moneymanagerex/wiki/GitCop

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: 33aafe54d018a00a536c6385e44422e814f21ba6
  * Invalid type. Valid types are **feat, fix, docs, style, refactor, perf, test, chore, revert, cleanup, format**

Guidelines are available at https://github.com/moneymanagerex/moneymanagerex/wiki/GitCop

---
This message was auto-generated by https://gitcop.com

File: src/mmOptionViewSettings.cpp [hunk 1]
```diff
@@ -67,7 +67,7 @@ void mmOptionViewSettings::Create()
         , wxDefaultPosition, wxDefaultSize, itemChoiceViewAccountTranslatedStrings);
     view_sizer1->Add(m_choice_visible, g_flags);
 
-    wxString vAccts = Model_Setting::instance().ViewAccounts();
+    const wxString& vAccts = Model_Setting::instance().ViewAccounts();
```
Comment (guanlisheng) [inline src/mmOptionViewSettings.cpp]: this is incorrect

**Comment by guanlisheng** [comment]
please send pull request per one single subject and this PR includes too many other changes.

**Pull request closed without merging**